support-frontend/package.json

```diff
@@ -92,7 +93,6 @@
 		"@sentry/browser": "^8.33.1",
 		"@stripe/react-stripe-js": "^2.7.3",
 		"@stripe/stripe-js": "^4.1.0",
-		"@types/dompurify": "^3.0.5",
```

Comment:
There were a couple of unused dependencies which knip is now picking up but which it wasn't before - again not 100% sure why!